pages/learn/integration/pre-built-integrations/ftp-edi-message-to-salesforce-opportunity.js

```diff
@@ -315,15 +315,15 @@ export default function Learn({ samples, content }) {
                   <Col xs={12} lg={6} style={{ fontSize: "18px" }}>
                     <p>Interactions between businesses, such as sending purchase orders and invoices, usually occur over
                       EDI-based B2B channels. On the other hand, sales and customer details related to such transactions
-                      are maintained in Salesforce. Therefore, it&apos;s critical to bridge salesforce with B2B channels in
-                      order to automate the sales processes. For example, it is It is possible to update the status and
-                      details of opportunities in Salesforce based on the exchange of EDI messages like EDIFACT REQOTE
-                      (Request for quotation), EDIFACT QUOTES (Quotation response), EDIFACT ORDERS (Purchase order), etc.
+                      are maintained in Salesforce. Therefore, it&apos;s critical to bridge Salesforce with B2B channels in
+                      order to automate the sales processes. For example, it is possible to update the status and
+                      details of opportunities in Salesforce based on the exchange of EDI messages like `EDIFACT REQOTE
+                      (Request for quotation)`, `EDIFACT QUOTES (Quotation response)`, `EDIFACT ORDERS (Purchase order)`, etc.
```

Comment:
Do we need backticks here as those are not variables or constants in our side? Btw fine to have, then we should have it only around  EDIFACT REQOTE and EDIFACT QUOTES and not around the brackets followed by.

Reply:
Fixed.